### ISSUE TITLE
[Menu design][Accessibility] Save menu container is missing

### ISSUE THREAD

**Issue by natastro** [mozilla-mobile/firefox-ios#23468]
## Prerequisites

- Turn on VoiceOver
- Turn on treatment-a of the iOS Menu Redesign
- Found on Firefox Beta iOS app 134.0
- Open any webpage

## Steps to reproduce

1. Open Firefox Beta iOS app
2. Open the menu from the hamburger menu icon button at the bottom of the screen
3. As the action sheet appears on screen, navigate to the Save submenu and double tap to activate
4. Navigate to the close 'x' button at the top right corner. Then swipe right to move to the next item in VoiceOver's focus, observe speech output

### Expected behavior / Actual behavior

A menu named 'Save' should be in the focus order, after the Close 'x' button according to the Accessibility Reading Order in this [file](https://www.figma.com/design/RFz9fYtotQCQuinwcZujZt/%5B%5D-Menu-Redesign?node-id=19728-177911&t=srxir48oA5EvwzbP-4)

### Device & build information
* Device: iPhone 15, iOS 18.1.1
* Build version: Firefox Beta 134.0
* First seen version: First time I've seen this issue
### Notes
Attachments: 

https://github.com/user-attachments/assets/dc255178-1fb7-40a8-80ad-1cc8c1bc7fab

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-10751)

**Comment by data-sync-user** [comment]
➤ Natalie Stroud commented:

Canceling- particular menu doesn’t exist. Individual buttons should be their own focus stops
